app/components/public/side-menu.js

```diff
@@ -24,7 +23,7 @@ export default class SideMenu extends Component {
   }
 
   async checkSpeakers() {
-    this.showSpeakers = this.showSpeakers || (await this.loader.load(`/events/${this.event.id}/speakers?fields[speaker]=id&page[size]=1&filters=${JSON.stringify(SPEAKERS_FILTER)}`)).data.length;
+    this.showSpeakers = this.showSpeakers || this.event.speakers.length;
```

Comment:
As I said, this is not a good idea as it loads all speakers in one go. Secndly, it does not filter the speakers who have no accepted or confirmed sessions. It was used for a reason

Reply:
trying to figure out why it's happening there.

Reply:
https://api.eventyay.com//v1/events/1211/speakers and https://api.eventyay.com//v1/events/5ac8e4d9/speakers they both are on same page.
first one is returning empty response and second is giving response

Reply:
There must be some discrepancies in filters used